plugin/pages/include/control.php

```
@@ -96,6 +96,37 @@ function build_response(int $rc, array $out) {
+            echo json_encode(['error' => "enabled must be 'yes' or 'no'"]);
+            break;
+        }
+        // Line-based upsert so every other key in vault.cfg is preserved.
+        $lines = [];
+        if (file_exists($CONFIG)) {
+            $raw = @file_get_contents($CONFIG);
+            if ($raw !== false && $raw !== '') {
+                $lines = preg_split('/\r?\n/', rtrim($raw, "\r\n"));
+            }
+        }
+        $lines = array_values(array_filter($lines, function ($l) {
```

PHPStan (2.2.2)

[warning] Parameter `#1` $array of function array_filter expects array, list<string>|false given.


(argument.type)